auth_session_timeout/README.rst

```diff
@@ -13,16 +15,32 @@ Configuration
 
 Two system parameters are available:
 
-* inactive_session_time_out_delay: validity of a session in seconds (default = 2 Hours)
-* inactive_session_time_out_ignored_url: technical urls where the check does not occur
+* ``inactive_session_time_out_delay``: validity of a session in seconds
+  (default = 2 Hours)
+* ``inactive_session_time_out_ignored_url``: technical urls where the check
+  does not occur
+
+Usage
+=====
+
+Setup the session parameters as described above.
```

Comment:
I thought about it, but I'm honestly torn. They are more of a configuration instruction vs usage. Maybe instead I add `Use system like normal` in place of the note to configure using instructions?